src/plugin/services/InternalFileWriter.ts

```
@@ -0,0 +1,445 @@
+  }
+
+  /**
+   * Get the plugin base path
+   */
+  private getPluginBasePath(): string {
+    if (typeof window !== 'undefined') {
+      // In browser environment, we'll use the Vite plugin API
+      return '/src/plugin/available';
+    } else {
+      // In Node.js environment (SSR, build), use actual path
+      const path = require('path');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)